trunk/src/lsc/externaldata.py

```diff
@@ -420,17 +424,20 @@ def sdss_swarp(imglist,_telescope='spectral',_ra='',_dec='',output='', objname='
     ar = np.where(ar2 == 0, _saturate, ar)
 
     if len(skylevel):
-        hd['SKYLEVEL'] = (np.mean(skylevel), 'average sky level')
+        hd['SKYLEVEL'] = (np.nanmean(skylevel), 'average sky level')
     hd['L1FWHM']   = (9999,       'FHWM (arcsec) - computed with sextractor')
     hd['WCSERR']   = (0,          'error status of WCS fit. 0 for no error')
-    hd['MJD-OBS']  = (_mjd,       'MJD')
+    hd['MJD']      = (_mjd,       'MJD')
+    hd['MJD_OBS']  = (_mjd,       'MJD')
```

Comment:
Why did you change these keywords? The standard keyword for LCO images is MJD-OBS.

Reply:
Megan says:
> The header keywords "MJD"/"MJD_OBS" needed to be changed to those values because we were getting a keyword error when using "MJD-OBS" otherwise. This has been confirmed by running with the old keyword header and getting an error of keyword not found.

The issue I'm worried about is that now you have *removed* `MJD-OBS` for the SDSS images. Have you tested that this branch of the code does not break SDSS ingestion?

Reply:
I confirmed that `MJD-OBS` is the keyword recognized by the pipeline, so I reverted this change.